README.md

```diff
@@ -9,19 +9,28 @@
 
 ### Install Prerequisites
 #### Install Git
- * If you want to use Git in the command line, run `sudo apt install git`
+ * If you want to use Git in the command line, run `sudo apt install git` on Ubuntu, on Windows you can download [Git for Windows](https://git-scm.com/) or use `choco install git` if you have Chocolatey installed.
  * If you're less terminal-inclined, you can install a Git GUI wrapper. [There's a nice list here.](https://git-scm.com/downloads/guis/)
     * I would personally recommend [GitKraken](https://www.gitkraken.com/). The free version has everyhting you need, (although their conflict resolver for Pro is nice,) and you can also get Pro free with [GitHub Education](https://education.github.com/)!
+#### Install CMake
+ * Debian-based (Ubuntu) run `sudo apt install cmake`
+ * Fedora/RHEL-based `sudo dnf install cmake`
+ * Windows Download [CMake](https://cmake.org/download/) or run `choco install cmake` if you have Chocolatey installed.
+#### Install vcpkg
+ * Vcpkg is a cross platform library manager, you can install it by reading the instructions on this [page](https://github.com/microsoft/vcpkg)
+ * This program is required to build echo-bot under Windows.
```

Comment:
Refer to Echo with the proper capitalized name at the top of the repo, not the constriined-formatting repo name